VTEX - Giftcard API.json

```diff
@@ -1253,153 +1253,159 @@
+                        "description": "Represents the relationship between the client and the store."
+					},
+					"redemptionCode": {
+						"type": "string",
+						"default": "AH52F7",
+						"description": "Minimum of six characters"
+					},
+					"discounts": {
+						"type": "integer",
+						"format": "int32",
+                        "description": "Discounts value."
+					},
+					"shipping": {
+						"type": "integer",
+						"format": "int32",
+                        "description": "Shipping value."
+					},
+					"taxes": {
+						"type": "integer",
+						"format": "int32",
+                        "description": "Taxes value."
+					},
+					"items": {
+						"type": "array",
+						"items": {
+							"$ref": "#/components/schemas/Item"
+						},
+						"description": ""
+					},
+					"itemsTotal": {
+						"type": "integer",
+						"format": "int32",
+                        "description": "Total items value."
+					}
+				},
+				"default": {
+					"grandTotal": 182,
+					"relationName": null,
+					"redemptionCode": "XTYB",
+					"discounts": 20,
+					"shipping": 2,
+					"taxes": 0,
+					"items": [
+						{
+							"productId": "2000000",
+							"id": "2000002",
+							"refId": "MEV41",
+							"name": null,
+							"price": 200,
+							"quantity": 1
+						}
+					],
+					"itemsTotal": 200
+				}
+			},
             "Item": {
-                "title": "Item",
-                "required": [
-                    "productId",
-                    "id",
-                    "refId",
-                    "name",
-                    "price",
-                    "quantity"
-                ],
-                "type": "object",
-                "properties": {
-                    "productId": {
-                        "type": "string",
-                        "default": "5"
-                    },
-                    "id": {
-                        "type": "string",
-                        "default": "5"
-                    },
-                    "refId": {
-                        "type": "string",
-                        "default": "12"
-                    },
-                    "name": {
-                        "type": "string",
-                        "default": "Insert name here"
-                    },
-                    "price": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 50
-                    },
-                    "quantity": {
-                        "type": "integer",
-                        "format": "int32",
-                        "default": 1
-                    }
-                },
-                "example": {
-                    "productId": "1",
-                    "id": "1",
-                    "refId": "12",
-                    "name": "Product Name",
-                    "price": 100,
-                    "quantity": 1
-                }
-            },
+				"title": "Items information.",
```

Comment:
This component is used in a request body, therefore it must abide by the OAS guidelines for request bodies, please include examples to each field.

See our [openapi template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc)

Reply:
Corrigido.